tick/stateful/expr_test.go

```diff
@@ -1321,10 +1334,10 @@ func runCompiledEvalTests(
 	}
 }
 
-func mustCompileExpression(t *testing.T, node tick.Node) stateful.Expression {
+func mustCompileExpression(node tick.Node) stateful.Expression {
 	se, err := stateful.NewExpression(node)
 	if err != nil {
-		t.Fatalf("Failed to compile expression: %v", err)
+		panic(fmt.Sprintf("Failed to compile expression: %v", err))
```

Comment:
I changed it from panic so that you get an ugly stack trace ;)  The stack trace makes it easier to figure out which expression didn't compile. Otherwise you have to look through all the expressions in a test. 

Also it should only fail when you are changing a test and accidentally create an expression that fails to compile. If its possible for a given expression not to compile then don't use this method and check the error explicitly.